packages/salesforcedx-sobjects/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@salesforce/salesforcedx-sobjects",
+  "displayName": "Salesforce Faux SObject Generator",
+  "description":
+    "Fetches sobjects and generates their faux apex class to be used for Apex Language Server",
+  "version": "40.8.0",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.15.0"
+  },
+  "dependencies": {
+    "@salesforce/salesforcedx-utils-vscode": "40.8.0",
+    "jsforce": "1.7.1"
```

Comment:
Ensure that this is the same version used by the CLI. And then also add @types/jsforce to the devDependencies section.